rules/rules-reviewed/cloud-readiness/embedded-cache-libraries.windup.xml

```diff
@@ -289,13 +289,34 @@
                 </or>
             </when>
             <perform>
-                <classification title="Caching - Spring Boot Cache library" category-id="information" effort="0">
+                <classification title="Caching - Spring Boot Cache library" category-id="cloud-mandatory" effort="5">
```

Comment:
Isn't worth having this to become a `hint` considering the content it provides?
It looks more than a "simple" classification but rather a suggestion on how to migrate the embedded cache to an external caching service.

Reply:
@mrizzi - What is the benefit of  a hint over a classification? These issues don't have any supporting links. And they appear on the Issue report fine.

Reply:
Ah, not a practical benefit on the reports but rather more a formal change.